sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any
+
+from azure.core import AsyncPipelineClient
+from msrest import Deserializer, Serializer
+
+from .._models import CloudEvent, EventGridEvent
+from azure.core.pipeline.policies import AzureKeyCredentialPolicy
+from azure.core.credentials import AzureKeyCredential
+from .._generated.event_grid_publisher_client.aio._event_grid_publisher_client_async import EventGridPublisherClient as EventGridPublisherClientAsync
```

Comment:
from .._generated.event_grid_publisher_client.aio import EventGridPublisherClient as EventGridPublisherClientAsync

Don't use a a private part when not needed